### PR TITLE
Update tldraw version in tldraw demo

### PR DESCRIPTION
**Pull request by paulgb**
Demo: https://y-sweet-demos-git-paulgb-update-tldraw-version-drifting-corp.vercel.app/tldraw?doc=o1rclg2-Brc0G-ECrnojc

### CODE REVIEW

**Review by pretentious7 — APPROVED** [review]

**Review by rolyatmax — APPROVED** [review]

File: examples/nextjs/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -89,10 +89,9 @@ export default function Sidebar({ children }: { children: React.ReactNode }) {
   return (
     <>
       <div className="h-screen w-full absolute">
-        <Transition.Root show={sidebarOpen} as={Fragment}>
```
Comment (rolyatmax) [inline examples/nextjs/src/components/Sidebar.tsx:92]: Curious what prompted this change.

Comment (paulgb) [reply examples/nextjs/src/components/Sidebar.tsx:92]: Some types changed, I don't know what the background is.

image

Link to full logs: https://vercel.com/drifting-corp/y-sweet-demos/9x7BCmmzLLQTazGuLwS8ohFeTqvh

Comment (rolyatmax) [reply examples/nextjs/src/components/Sidebar.tsx:92]: It makes the mobile menu look like this:

Screenshot 2024-01-12 at 1 14 14 PM

But it should look like this:

Screenshot 2024-01-12 at 1 14 07 PM

Comment (paulgb) [reply examples/nextjs/src/components/Sidebar.tsx:92]: Good catch. I'll see if there's anything I can do.

**Pull request merged**